lldb/source/API/SBValue.cpp

```diff
@@ -1270,6 +1270,11 @@ bool SBValue::GetDescription(SBStream &description) {
     DumpValueObjectOptions options;
     options.SetUseDynamicType(m_opaque_sp->GetUseDynamic());
     options.SetUseSyntheticValue(m_opaque_sp->GetUseSynthetic());
+    if (short_mode) {
```

Comment:
Is any of this affected by the `dwim-print-verbosity` setting?
Should we print the value using `dwim-print`? I'm not sure if this is different than `dwim-print`.